project/Http4sPlugin.scala

```diff
@@ -29,6 +29,8 @@ object Http4sPlugin extends AutoPlugin {
   val scala_212 = "2.12.17"
   val scala_3 = "3.2.0"
 
+  private val CompileTime = config("compile-time").hide
```

Comment:
... another thing to upstream 😂 

Awesome though, thanks for figuring this out!!

Reply:
Yes, I think it could be added to sbt-typelevel.
It does not look invasive – if not used, it should not cause any effect.